### DISCUSSION TITLE
240326

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#80] [팀회의]

**Comment by jisunipark** [comment]
## 논의사항

## 어제 한 일
- 사이드바 / 이슈 / 팀 백엔드 api 연동

## 오늘 할 일
- 자유게시판 ui 수정
- 자유게시판 api 연동
- 자유게시판 페이지네이션

## 오늘의 일정 (코어 타임)

## 멘토님 질문 사항

**Comment by MoonPillGyeom** [comment]
## 논의사항
x
## 어제 한 일
마지막으로 놀았습니다
## 오늘 할 일
바뀐 디자인 만들기
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항

**Comment by caseV12** [comment]
## 논의사항

## 어제 한 일
signup 완성, 멤버리스트 구경
## 오늘 할 일
멤버리스트...?
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항
